scripts/replication.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/env python
+# https://github.com/chambridge/replication_subscriber
```

Comment:
can you add the hash the file was pulled from? even if it's in a new line or something

Reply:
Oh, good idea.